test/packages/parallel/aws/data_stream/ec2_metrics/_dev/deploy/tf/main.tf

```diff
@@ -2,7 +2,17 @@ variable "TEST_RUN_ID" {
   default = "detached"
 }
 
-provider "aws" {}
+provider "aws" {
+  default_tags {
+    tags = {
+      Environment = "CI"
+      Owner       = "elastic-package"
```

Comment:
Let's pull those too from `var.*`. Otherwise, developers may forget to populate them or misuse.